vm/actor/tests/common/mod.rs

```diff
@@ -161,10 +161,10 @@ impl<'a, BS: BlockStore> MockRuntime<'a, BS> {
                 actor::paych::Actor.invoke_method(self, method_num, params)
             }
             x if x == &*MULTISIG_ACTOR_CODE_ID => {
-                actor::cron::Actor.invoke_method(self, method_num, params)
+                actor::multisig::Actor.invoke_method(self, method_num, params)
             }
             x if x == &*REWARD_ACTOR_CODE_ID => {
-                actor::cron::Actor.invoke_method(self, method_num, params)
+                actor::reward::Actor.invoke_method(self, method_num, params)
```

Comment:
what a lapse this was missed, good catch! I assume this code block was copied from the default runtime because this also exists there, do you mind fixing there was well?